projects/shared/src/lib/components/split-button/split-button-option.model.ts

```diff
@@ -1,4 +1,5 @@
 export interface SplitButtonOptionModel {
   id: string;
   label: string;
+  webMercatorAvailable?: boolean;
```

Comment:
Should not be added here. This is a shared component, could be used anywhere (also not map related at all) so a map specific (webMercatorAvailable) property does not belong here. Much better to add for example a 'className' property which you can pass from the 'parent' component